Tools/cases_generator/instructions.py

```diff
@@ -209,7 +209,10 @@ def write_body(
                 space, cond = m.groups()
                 space = extra + space
                 target = family.name if family else self.name
-                out.write_raw(f"{space}DEOPT_IF({cond}, {target});\n")
+                if tier == TIER_TWO:
+                    out.write_raw(f"{space}TIER2_DEOPT_IF({cond}, {target});\n")
+                else:
+                    out.write_raw(f"{space}DEOPT_IF({cond}, {target});\n")
```

Comment:
Isn't this the wrong place to fix this? There's already a redefined DEOPT_IF() in ceval.c around L959.

Reply:
Okay, I resolved the other conversation, but I'm still curious about this one.

Reply:
Sorry I have some changes on my end that I havent pushed yet. This fixes it.